functions.js

```diff
@@ -0,0 +1,107 @@
+//Task 1. Напишите функцию, которая принимает 2 числа и возвращает 1, если первое число больше, чем второе; 
+//-1, если первое число меньше, чем второе, и 0, если числа равны.
+//Попробуйте реализовать эту функцию не только с помощью if...else, 
+//но и с помощью тернарного оператора, и в виде стрелочной функции.
+//const num1 = +prompt('Enter a number');
+//const num2 = +prompt('Enter one more number');
+
+//function numberComparison(num1, num2) {
+//  if (num1 > num2) {
+//    return '1';
+//  }else if (num1 < num2) {
+//    return '-1';
+//  }else {
```

Comment:
you can remove last if for a better readibility

Reply:
why? the task specifies that if the numbers are equal return 0

Reply:
fixed